### ISSUE TITLE
cnn 코드 다양하게 구성해서 돌려보는 중. 돌리는데 시간이 꽤 걸리고 부하가 커서 learning rate 등의 parameters 는 바꾸지 않고 window 만 다르게 구성해봤음. 각각의 경우 정확도 기록함.

### ISSUE THREAD

**Issue by smallerhand** [1eedaegon/KYLius-method#30]
<parameters >
learning_rate = 0.001
training_epochs = 15
batch_size = 100
1) 
window = 3*3, 32개/ 3*3, 64개
정확도 98.57%
2)
window = 4*4, 32개/ 4*4, 64개
정확도 98.48%/ 98.36%/ 98.56%
3)
정확도가 낮아질 것 같긴 하지만 비교삼아 윈도우 개수를 적게 해봄
window = 5*5, 10개/ 3*3, 5개
정확도 97.48%
4) 
3에서 병렬로 구성, window 크기를 다르게 L1_2랑 L2_2 만듬.
w1, w2= 5*5, 10개/ 3*3, 5개
w1_2, w2_2 = 3*3, 12개/ 3*3, 5개
정확도 98.09%
(3번보다 정확도 높아졌으므로 이 구조에서 window개수를 다시 늘리면 더 높아질 듯.)
5)
병렬로 구성하고 window 크기도 원래만큼 늘림.
w1, w2= 5*5, 32개/ 3*3, 64개
w1_2, w2_2 = 3*3, 36개/ 3*3, 40개
정확도 98.48%
생각보다 높지 않았다. 
*개선할 사항:
윈도우 개수랑 병렬 다시 조절해보기.
dropout 추가해보기.
